### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by crimist**
* Add ampere verification for direct backend to docs
* Minor restructuring to improve clarity
* Add links to improve clarity

### CODE REVIEW

**Comment by crimist** [comment]
Does the direct backend still need the experimental warning etc? It does seem like very few people are having issues with it so I could clean it up and just add an line that says "here's an issue template if you have a problem". You're more familiar with the issue tracker though so lmk.

File: README.md [hunk 1]
```diff
@@ -22,7 +21,7 @@ This library requires that the `nvidia_drm` kernel module is [configured with th
 
 # Codec Support
 
-Only decoding video is currently supported.
+Hardware encoding only, decoding is [not supported](/../../issues/116).
```
Comment (elFarto) [inline README.md:24]: You've switched the meaning here. Only decoding is supported, not encoding.

File: README.md [hunk 2]
```diff
@@ -125,10 +128,11 @@ a greater degree of control over buffer allocation and freeing.
 
 Given this backend accesses the NVIDIA driver directly, via NVIDIA's unstable API, this module is likely to break often with new versions of the kernel driver. If you encounter issues using this backend raise an issue and including logs generated by `NVD_LOG=1`.
 
-**Currently tested and verified generations**
+**Verified working generations**
 
 * Pascal (10XX)
-* Turing mobile (20XX)
+* Turing (20XX)
+* Ampere (30XX)
```
Comment (elFarto) [inline README.md:135]: Looks like we've also got a report of it working from an A4000, so we could add that here.

**Comment by elFarto** [comment]
> Does the direct backend still need the experimental warning etc?

I think so. While it does appear to be working for everyone, it could easily break on the next driver upgrade. It's also not the NVIDIA supported way to do this, and relies on me guessing how the NVIDIA driver API works.

File: README.md (hunk 1 above)
Comment (crimist) [reply README.md:24]: Good catch 🤦🏻

**Comment by crimist** [comment]
Ah I see, so it uses undocumented functions?

**Comment by elFarto** [comment]
> Ah I see, so it uses undocumented functions?

That's correct.

**Comment by elFarto** [comment]
Thanks for the changes!

**Pull request merged**